src/sentry/static/sentry/app/views/settings/organization/general/organizationSettingsForm.jsx

```diff
@@ -22,11 +23,25 @@ const NewOrganizationSettingsForm = React.createClass({
     onSave: PropTypes.func.isRequired,
   },
 
-  mixins: [ApiMixin],
+  mixins: [ApiMixin, OrganizationState],
 
   render() {
     let {initialData, orgId, onSave} = this.props;
 
+    //Only for adding the Flag to 2FA Enforcement.
+    if (this.getFeatures().has('require-2fa')) {
+      let security_panel = organizationSettingsFields.find(
```

Comment:
this global mutation will cause problems if require-2fa is ever changeable by the user or by switching between orgs. we need to be careful and replace this code with a better approach when we change how the feature flag works

Reply:
@MaxBittker the require-2fa is a config level switch. If we ever need to change it to org-level it will need to be rewritten. The code will be removed once the feature is finished, and then require-2fa will be like any other option. I need to make a corresponding addition to getsentry to set it for internal users only.